palace/drivers/basesolver.cpp

```diff
@@ -84,6 +89,282 @@ BaseSolver::BaseSolver(const IoData &iodata, bool root, int size, int num_thread
   }
 }
 
+std::string BaseSolver::IterationPostDir(int iter) const
```

Comment:
Seems like this can be in an anonymous namespace like the `GetPostDir` above just to be consistent.